--- popt-1.7/configure.in.wiget	Thu Sep 19 02:09:30 2002
+++ popt-1.7/configure.in	Thu Sep 19 02:14:19 2002
@@ -5,7 +5,7 @@
 AM_INIT_AUTOMAKE(popt, 1.7)
 AM_CONFIG_HEADER(config.h)
 
-ALL_LINGUAS="cs da de es eu_ES fi fr gl hu id is it ja ko no pl pt pt_BR ro ru sk sl sr sv tr uk wa zh zh_CN.GB2312"
+ALL_LINGUAS="cs da es gl hu is ko no pl pt ro ru sk sl sv tr uk wa zh_CN.GB2312"
 
 AC_ISC_POSIX
 
@@ -86,4 +86,4 @@
	[Full path to popt top_sourcedir.])
 AC_SUBST(POPT_SOURCE_PATH)
 
-AC_OUTPUT([Doxyfile Makefile intl/Makefile po/Makefile.in])
+AC_OUTPUT([Doxyfile Makefile intl/Makefile po/Makefile.in])
--- popt-1.7/Makefile.am.wiget	Thu Sep 19 02:09:38 2002
+++ popt-1.7/Makefile.am	Thu Sep 19 02:13:02 2002
@@ -2,12 +2,12 @@
 
 AUTOMAKE_OPTIONS = 1.4 foreign
 
-EXTRA_DIST = autogen.sh CHANGES $(man_MANS) popt.spec \
+EXTRA_DIST = config.rpath autogen.sh CHANGES $(man_MANS) popt.spec \
 	testit.sh test-poptrc \
 	po/*.in po/*.po po/popt.pot \
 	popt.ps
 
-SUBDIRS = intl po
+SUBDIRS = intl  po
 
 INCLUDES = -I. -I$(top_srcdir)
 
@@ -74,3 +74,5 @@
 	rm -rf doxygen
 	mkdir -p doxygen
 	doxygen
+
+ACLOCAL_AMFLAGS = -I m4
