Index: popt.c
===================================================================
RCS file: /v/rpm/cvs/popt/popt.c,v
retrieving revision 1.101
retrieving revision 1.103
diff -u -u -r1.101 -r1.103
--- popt.c	14 Jun 2007 13:31:10 -0000	1.101
+++ popt.c	22 Jun 2007 00:02:01 -0000	1.103
@@ -396,7 +396,7 @@
     if (!strchr(item->argv[0], '/') && con->execPath != NULL) {
 	char *s = malloc(strlen(con->execPath) + strlen(item->argv[0]) + sizeof("/"));
 	if (s)
-	    sprintf(s, "%s/%s", con->execPath, item->argv[-1]);
+	    sprintf(s, "%s/%s", con->execPath, item->argv[0]);
 	argv[argc] = s;
     } else
 	argv[argc] = POPT_findProgramPath(item->argv[0]);
@@ -764,7 +764,7 @@
 
 	/* Process next long option */
 	if (!con->os->nextCharArg) {
-	    char * optString;
+	    const char * optString;
             int optStringLen;
 	    int thisopt;
 
@@ -805,7 +805,7 @@
 		con->restLeftover = 1;
 		continue;
 	    } else {
-		char *oe;
+		const char *oe;
 		int singleDash;
 
 		optString++;
Index: poptint.c
===================================================================
RCS file: /v/rpm/cvs/popt/poptint.c,v
retrieving revision 1.4
retrieving revision 1.8
diff -u -u -r1.4 -r1.8
--- poptint.c	14 Jun 2007 07:54:52 -0000	1.4
+++ poptint.c	15 Jun 2007 13:50:12 -0000	1.8
@@ -97,9 +97,14 @@
 {
   char *buffer = NULL;
   char c;
+  va_list apc;
+
+  va_copy(apc, ap);	/* XXX linux amd64/ppc needs a copy. */
 
   buffer = calloc (sizeof (char), vsnprintf (&c, 1, format, ap) + 1);
-  vsprintf (buffer, format, ap);
+  vsprintf (buffer, format, apc);
+
+  va_end(apc);
 
   return buffer;
 }
@@ -136,7 +141,6 @@
     retval = fprintf (stream, "%s", locale_str);
     free (locale_str);
   } else {
-    fprintf (stderr, POPT_WARNING "%s\n", "Invalid UTF-8");
 #endif
     retval = fprintf (stream, "%s", buffer);
 #ifdef HAVE_ICONV
Index: poptint.h
===================================================================
RCS file: /v/rpm/cvs/popt/poptint.h,v
retrieving revision 1.24
retrieving revision 1.25
diff -u -u -r1.24 -r1.25
--- poptint.h	25 May 2007 17:36:23 -0000	1.24
+++ poptint.h	15 Jun 2007 12:49:56 -0000	1.25
@@ -114,8 +114,6 @@
 
 #define N_(foo) foo
 
-#define POPT_WARNING "(popt): Warning **: "
-
 int   POPT_fprintf (FILE* steam, const char *format, ...);
 char *POPT_prev_char (const char *str);
 
