diff -urN popt-1.10.7/configure.ac popt-1.10.7.new/configure.ac
--- popt-1.10.7/configure.ac	2006-04-09 20:19:27.000000000 +0200
+++ popt-1.10.7.new/configure.ac	2006-11-27 17:21:57.000000000 +0100
@@ -14,7 +14,7 @@
 
 AM_INIT_AUTOMAKE([foreign])
 
-ALL_LINGUAS="cs da de es eu_ES fr gl hu is ja ko nl no pl pt ro ru sk sl sv tr uk vi wa"
+ALL_LINGUAS="cs da de es eu fr gl hu is ja ko nl nb pl pt ro ru sk sl sv tr uk vi wa"
 
 AC_PROG_CC
 AC_PROG_INSTALL

