diff -urN popt-1.10.7/configure.ac popt-1.10.7.new/configure.ac
--- popt-1.10.7/configure.ac	2006-04-09 20:19:27.000000000 +0200
+++ popt-1.10.7.new/configure.ac	2006-11-27 17:21:57.000000000 +0100
@@ -13,7 +13,7 @@
 AC_SUBST(LT_REVISION, 0)
 AC_SUBST(LT_AGE, 0)
 
-ALL_LINGUAS="cs da de es eu_ES fi fr gl hu id is it ja ko no pl pt pt_BR ro ru sk sl sr sv tr uk wa zh zh_CN zh_TW"
+ALL_LINGUAS="cs da de es fi fr gl hu id is it ja ko nb pl pt pt_BR ro ru sk sl sr sv tr uk wa zh zh_CN zh_TW"
 
 AC_ISC_POSIX
 

